server/methods/browseChannels.js

```diff
@@ -132,6 +132,13 @@ const getTeams = (user, searchTerm, sort, pagination) => {
 		roomsCount: getChannelsCountForTeam(room.teamId),
 	}));
 
+	if (sort.channelsCount) {
+		rooms.sort((a, b) => {
```

Comment:
This is probably nit picking, but, can you move the callback to a named function? So others can know from the name what this is doing 🙈

Reply:
Done!